build.gradle

```diff
@@ -49,13 +49,19 @@ idea {
 }
 
 // ### Java plugin settings
-sourceCompatibility = JavaVersion.VERSION_11
+sourceCompatibility = JavaVersion.VERSION_14
 
+compileJava.dependsOn(clean)
 compileJava {
     options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
 }
 
 test {
+    moduleOptions {
+        addExports = ["org.junit.platform.commons/org.junit.platform.commons.util": "ALL-UNNAMED",
+                      "org.junit.platform.commons/org.junit.platform.commons.logging": "ALL-UNNAMED"]
+    }
+    maxHeapSize = "2056m"
```

Comment:
woah.  what does this mean?

Reply:
So the moduleOptions setting was necessary for tests to actually run after doing the upgrades, and the maxHeapSize (I determined through trial and error) is what allows the big tests to actually finish running and not crap out (i.e. the fix for #283).